docs/user_guide/quickstart.rst

```diff
@@ -20,58 +20,12 @@ By default, SiliconCompiler will send remote jobs to our public beta servers, af
 
```

Comment:
The first line of "3.1 Remote Run" now has a broken link. Direct the _private_ link to :ref:`Configuring a Different Remote Server`

Reply:
I believe that not all of these links will work properly until #1365 is merged.